qemu/tests/cfg/virtio_fs_multi_users_access.cfg

```diff
@@ -0,0 +1,58 @@
+- virtio_fs_multi_users_access:
+    type = virtio_fs_multi_users_access
+    filesystems = fs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir = virtio_fs_test/
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    driver_name = viofs
+    fs_target = 'myfs'
+    fs_dest = "/mnt/${fs_target}"
+    fs_driver_props = {"queue-size": 1024}
+    mem = 4096
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = 4G
+    use_mem_mem1 = no
+    share_mem = yes
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    # install winfsp tool
+    i386, i686:
+        install_winfsp_path = 'C:\Program Files'
+    x86_64:
+        install_winfsp_path = 'C:\Program Files (x86)'
+    virtio_win_media_type = iso
+    cdroms += " virtio"
+    Windows:
+        cmd_dd = 'dd if=/dev/random of=%s bs=1M count=100'
+    variants:
+        - localgroup_users:
+            new_user = "user1 user2"
+            new_pwd = "U1pwd345?a U2pwd345?a"
+            add_user_cmd = "useradd -m -p `openssl passwd -1 -salt '29' %s` %s"
+            del_user_cmd = "userdel -r -f %s"
+            Windows:
+                add_user_cmd = "net user %s %s /add"
+                del_user_cmd = "net user %s /delete"
+        - domain_users:
+            only Windows
+            nics += " nic2"
+            nic_model_nic2 = virtio
+            netdst_nic2 = switch1
+            # Before test, the domain server has to be set exactly.
+            # This case only does the work on client side.
```

Comment:
hi @nickzhq , we may highlight these at the beginning of the cfg file, e.g. 

Please prepare the following before running virtio_fs_multi_users_access.domain_users:
1. Set up the domain server
2. Set up the switch1 bridge, used for accessing the domain server
(Not sure if these 2 are correct and if there is any other requirement)

Because the inline comments look like regular comments, it's hard to catch people's attention.